client/resource_group/controller/controller.go

```diff
@@ -701,6 +757,37 @@ func (gc *groupCostController) handleRUTokenResponse(resp *rmpb.TokenBucketRespo
 	}
 }
 
+func (gc *groupCostController) applyBasicConfigForRUTokenCounters() {
+	for typ, counter := range gc.run.requestUnitTokens {
+		if !counter.limiter.IsLowTokens() {
+			continue
+		}
+		initCounterNotify(counter)
+		var cfg tokenBucketReconfigureArgs
+		fillRate := getRUTokenBucketSetting(gc.ResourceGroup, typ)
+		cfg.NewBurst = int64(fillRate.Settings.FillRate)
+		cfg.NewRate = float64(fillRate.Settings.FillRate)
+		failpoint.Inject("degradedModeRU", func() {
+			cfg.NewRate = 99999999
+		})
+		counter.limiter.Reconfigure(gc.run.now, cfg, resetLowProcess())
+	}
+}
+
+func (gc *groupCostController) applyBasicConfigForRawResourceTokenCounter() {
+	for typ, counter := range gc.run.resourceTokens {
+		if !counter.limiter.IsLowTokens() {
```

Comment:
maybe we can not skip it to make sure be in downgrade, or add log here.

Reply:
I want low-token buckets to enter degraded mode. It will log if successfully enter degraded mode